recipes/libxml/libxml2.inc

```diff
@@ -11,18 +11,39 @@ DEPENDS_HOST_OS = "libdl"
 DEPENDS_HOST_OS:HOST_LIBC_mingw = ""
 DEPENDS_${PN} += "libc libm ${DEPENDS_HOST_OS}"
 
+EXTRA_OECONF_PYTHON = "--with-python-install-dir=${libdir}"
+EXTRA_OECONF_PYTHON:machine = "--without-python"
+EXTRA_OECONF_PYTHON:sdk = "--without-python"
+
+EXTRA_OECONF_DOC = "--with-catalog --with-docbook"
+EXTRA_OECONF_DOC:machine = ""
+
 EXTRA_OECONF = "\
  --without-legacy \
- --with-catalog \
- --with-docbook \
- --without-python \
- --with-c14n"
+ ${EXTRA_OECONF_DOC} \
+ --with-c14n \
+ ${EXTRA_OECONF_PYTHON}"
 
 LDFLAGS_LDL = "-ldl"
 LDFLAGS_LDL:TARGET_OS_mingw32 = ""
 export LDFLAGS += "${LDFLAGS_LDL}"
 
-PACKAGES =+ "${PN}-utils"
+DO_INSTALL_PYTHONFIXUP = "do_install_pythonfixup"
+DO_INSTALL_PYTHONFIXUP:machine = ""
+DO_INSTALL_PYTHONFIXUP:sdk = ""
+
+#This hack is needed python modules are not allowed to be in the same dir as the module
+#Libtool prevents the python modules being installed elsewhere
+do_install[postfuncs] += "${DO_INSTALL_PYTHONFIXUP}"
```

Comment:
python modules are not allowed to stay in the same directory as libpython2.so...